rasa/shared/core/domain.py

```diff
@@ -1849,6 +1861,21 @@ def count_slot_mapping_statistics(self) -> Tuple[int, int, int]:
 
         return (total_mappings, custom_mappings, conditional_mappings)
 
+    def does_action_explicitly_need_domain(self, action_name: Text) -> bool:
+        """Assert if action has explicitly stated that it needs domain.
+
+        Args:
+            action_name: Name of the action to be checked
+
+        Returns:
+            True if action has explicitly stated that it needs domain.
+            Otherwise, it returns false.
+        """
+        return (
+            self._actions_which_explicitly_need_domain is not None
```

Comment:
this attribute is never None though, it's either an empty list or a list with strings.